syntax = "proto3";

package request_models;

message UseSupplyRequestModel {
	int32 player_id = 1;
}